src/wattsi.pas

```diff
@@ -1180,11 +1180,19 @@    TCrossReferences = record
          end
          else
          begin
-            Warn('Could not find ID ' + ID + ' for annotation that uses URLs:');
-            for Bug in Feature.Bugs do
-               Writeln('   ', Bug.URL);
-            if (Feature.CanIUseCode <> '') then
+            if (Variant <> vDEV) then
+            begin
+               Warn('Could not find ID ' + ID + ' for annotation that uses URLs:');
+               for Bug in Feature.Bugs do
+                  Writeln('   ', Bug.URL);
+               if (Feature.CanIUseCode <> '') then
+                  Writeln('   https://caniuse.com/#feat=', Feature.CanIUseCode);
+            end
+            else if (Feature.CanIUseCode <> '') then
```

Comment:
else if (Feature.CanIUseCode <> '') then

Nit: I personally prefer this `else if`-on-the-same-line style, but given the existing style throughout the source is to break after `else` and before `if`, would it be better to follow that style here?